### ISSUE TITLE
[BUG] Mobile - website scrolling unuseable after clicking mobile menu

### ISSUE THREAD

**Issue by juanri0s** [Rebuild-Black-Business/RBB-Website#341]
**Describe the bug**
On mobile/tablet screens, the scroll becomes disabled after clicking the mobile menu icon. 

**To Reproduce**
1. Go to RBB on mobile device (Apple Safari, Android Chrome tested) 
2. Click on mobile menu
3. Close mobile menu
4. Try to scroll

**Expected behavior**
I expect main content scrolling to be disabled when the menu opens but then is enabled when menu is closed 

**Actual Behavior**
Scrolling disabled after closing mobile menu

**Screenshots**

**Comment by juanri0s** [comment]
Seems it only happens when you first initially load the site. After clicking navigating to each page and using the mobile menu, scrolling seems somewhat normal.

**Comment by sonAndrew** [comment]
Yes I have seen this as well. Can I be assigned this task to fix?

**Comment by juanri0s** [comment]
@sonAndrew yupp! Let me know if you have any questions
